### PR TITLE
Complete blog post on organizing project folders and files -- Leeza Sergeeva

### PR DESCRIPTION
**Pull request by sergeevaleeza**
Create blog post on how to structure directories and name files while working on data analytics project. There is also a reference to version control (git, GitHub, commit messages) which I'll add links later to corresponding blog posts from other students.

### CODE REVIEW

File: _posts/2020-11-08-organizing-your-folders.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  "Guide to Organizing Projects"
+date:   2020-11-08 12:54:04 -0700
+categories: guide projects organization folder
+---
+Once you have an idea of what you want to do for your projects, it's important to organize your working directory. If your project is organized well and everything is documented, anyone who's unfamiliar with your project should be able to look at your files and understand what you did and why. Organize everything the way that if you come back to that project in couple months or years you will able to understand what you did and why.
```
Comment (sdverma89) [inline _posts/2020-11-08-organizing-your-folders.md:7]: This paragraph is straight forward, but could use some grammatical fixes.

File: _posts/2020-11-08-organizing-your-folders.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  * clean\
+* code\
+  * functions\
+    * 01_process-data.R
+    * 02_create-figures-using-processed-data.R
+  * archived\
+* output\
+  * data-exploration-figures\
+  * final-figures\
+  * descriptive-title-for-output-file-batch\
```
Comment (sdverma89) [inline _posts/2020-11-08-organizing-your-folders.md]: Lines 16-27 should end in forward slashes `/` rather than a back slash `\`.

**Review by sdverma89 — CHANGES_REQUESTED** [review]
It's a clear and concise blog, just needs some minor fixes.

**Comment by melt6457** [comment]
@sergeevaleeza nice job on this blog post! When you get a chance implement @sdverma89 recommendations and we will wait to merge this until the referenced blog posts are uploaded to the website. Once you implement the changes recommended by Sahil, I will add your point on canvas.

@sdverma89 nice job on this review.

**Review by melt6457 — COMMENTED** [review]
@sdverma89 @sergeevaleeza I wanted to leave a formal review of my own on here as well. Nice job on this. I liked that you plan to reference other blogs on the website. I believed Jordan's is up and you can add the link to that.

File: _posts/2020-11-08-organizing-your-folders.md (hunk 2 above)
Comment (sergeevaleeza) [reply _posts/2020-11-08-organizing-your-folders.md]: Windows uses backslashes for paths, while everything else seems to use forward slashes. I guess you have a Mac computer while I have a Windows computer. I'll go ahead and change the slashes.

**Comment by naupaka** [comment]
@melt6457 I think this is ready for another look over and potentially to merge

**Comment by melt6457** [comment]
@naupaka I've reviewed the PR, she was hoping to reference other posts on the website. I've assigned the point in canvas already, but I was waiting to merge until the final references were added. @sergeevaleeza if you don't plan on adding more references to blog posts from other students I will go ahead and merge it.

**Pull request merged**